docs/source/using/profiles.adoc

```diff
@@ -64,3 +83,27 @@ $ minishift profile delete PROFILE_NAME
 ----
 
 NOTE: The default profile 'minishift'  can not be deleted.
+
+[[example-workflows-to create-new-profiles]]
+== Example workflows to create a new profile
+
+To create a new profile foo with specific configurations, run:
+
+----
+$ minishift profile set foo
+$ minishift config set memory 4GB
+$ minishift config set cpus 2
+$ minishift addon enable anyuid
+$ minishift start
+----
+
+Alternate method to create the profile 'foo'
+
+----
+$ minishift --profile foo config set memory 4GB
+$ minishift --profile foo config set cpus 2
+$ minishift --profile foo addon enable anyuid
+$ minishift --profile foo minishift start
+----
+
+NOTE: once you have started multiple profiles, you should use xref:set-active-profiles[`profile set`] command to switch between profiles.
```

Comment:
captial O - Once

Reply:
"you can" instead of "you should". Maybe make it a TIP as well

Reply:
@hferentschik I intentionally used `you should` as I do not users to use `--profile` after the profile is created as we know it has some side effects.